python/tests/test_aac/plugins/root_schema_must_have_name/test_root_schema_must_have_name.py

```diff
@@ -93,35 +94,24 @@ def test_root_schema_has_name_language_error1(self):
             context.remove_definitions(definitions)
 
 
-    #  This test should trigger a LanguageError exception from the parse_and_load call
-    #  So the only pass here is throwing a LanguageError
-    def test_root_schema_has_name_language_error2(self):
-        with self.assertRaises(LanguageError):
-
-            context = LanguageContext()
-            definitions = context.parse_and_load(root_schema_without_name_language_error2)
-            result = root_schema_has_name(definitions[0].instance, definitions[0], context.get_definitions_by_name("Schema")[0].instance)
-            context.remove_definitions(definitions)
-
-
     #  This test should trigger a ParserError exception from the parse_and_load call
     #  So the only pass here is throwing a ParserError
-    def test_root_schema_has_name_parser_error3(self):
+    def test_root_schema_has_name_parser_error2(self):
```

Comment:
Missed this numbering aspect when this first got added. Think since they are separate error types, their numbering should be distinct. 
```suggestion
    def test_root_schema_has_name_parser_error1(self):
```